features/support/env.rb

```diff
@@ -1,4 +1,5 @@
 require 'coveralls'
+require 'database_cleaner'
```

Comment:
IF you need this (and I'm not sure you do, `cucumber/rails' might be requiring this already) you need to move it out of Coveralls ' way.